packages/validator-query/README.md

```diff
@@ -0,0 +1,57 @@
+# AMP-Toolbox Validator Query
+
+Makes it easier to query published AMP Validator rules and extract information
+about required markup and attributes for all AMP formats.
+
+## Usage
+
+Install via:
+
+```
+$ npm install @ampproject/toolbox-validator-query@canary
+```
+
+### Including the Module
+
+#### ES Module (Browser)
+
+```javascript
+import { load } from '@ampproject/toolbox-validator-query';
```

Comment:
Hm, I'm OK with renaming, but I have two problems with that:

-   `fetch` sounds like it misrepresents what that method does since it may not fetch anything, depending on the options provided.
-   What other methods would `ValidationRules` have? I'm kind of reluctant to expose the `AmpValidatorQuery` class itself since the end user shouldn't need to instantiate it on their own and I don't see any other functions we'd need. That said, I'm OK with having `ValidationRules` an object with a single method if you feel that makes sense.

Reply:
The reason I prefer to export a class is that it becomes easier to test:

Users will use: 
```
const validationRules = await ValidationRules.fetch()
```

For testing you can do:

```
const validationRules = new ValidationRules(fetchMock).fetch()
```

`fetch` was just a suggestion. What I like about it is that it makes it clear that this might perform a network request (as opposed to `ValidationRules.get()`.

Reply:
So, I don't see a reason to make the class public since testing is already possible by directly including (see `AmpValidatorRulesSpec.js`) - so this is the closest we can get to package-private constructor. The user can also specify a mock to the `load` function via the `rules` parameter (e.g. `load({ rules: { ... } })`). Exposing the class makes me feel like it can cause a lot of confusion.

Reply:
Even better if we don't need to export a class, however, we still should export an object in that case:

```
module.exports = {
  fetch: ... (or whatever name we decide on)
}
```

This is to keep the possibility to add more methods in the future and to be consistent with the other modules, e.g.:

```
const AmpOptimizer = require('@ampproject/toolbox-optimizer');

const ampOptimizer = AmpOptimizer.create();
```

Reply:
Done, renamed to `fetch` and ensured it's exported as an object (it was already, but I made it clear in the readme).